### PR TITLE
PM-563: Terms Of Service and Privacy Policy

### PR DESCRIPTION
**Pull request by mmv08**

### CODE REVIEW

File: src/components/InstallMetamask/index.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  <div className={cx('installMetamask')}>
+    <button className={cx('closeButton')} onClick={() => closeModal()} />
+    <img src={OlympiaIcon} alt="logo" style={logoStyle} />
+    <h3 className={cx('installText')}>Install MetaMask</h3>
+    <p className={cx('downloadText')}>
+      Metamask is not currently installed or detected.{' '}
+      <a className={cx('downloadLink')} href="https://metamask.io/" target="_blank" rel="noopener noreferrer">
+        Please download and install MetaMask
+      </a>{' '}
+      to start using Olympia.
```
[the remark this answers is not in the document]
Comment (mmvsha) [reply src/components/InstallMetamask/index.js]: @apanizo Yeah. I forgot to change the name. Thank you

File: src/components/AcceptTOS/index.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+  closeModal, tosAgreed, ppAgreed, initProviders,
+}) => {
+  const disabled = !ppAgreed || !tosAgreed
+  const login = () => {
+    initProviders()
+    closeModal()
+  }
+  return (
+    <div className={cx('acceptTOS')}>
+      <button className={cx('closeButton')} onClick={() => closeModal()} />
```
[the remark this answers is not in the document]
Comment (mmvsha) [reply src/components/AcceptTOS/index.js]: ahh!

**Comment by mmvsha** [comment]
@apanizo also you can test it here http://yummy-low.surge.sh/

**Comment by apanizo** [comment]
;) interesting, are you using my automatically deployment script?

**Comment by mmvsha** [comment]
@apanizo Wish I did 😆 Just surge cli

**Pull request merged**